docs/change-log.md

```diff
@@ -2,11 +2,22 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update November 11th, 2022
+
+- **New:** Added autocompletion of [placeholders](https://docs.humanitec.com/reference/concepts/app-config/placeholders) in `labels`, `variables` and `files` properties of a workload in [Deployment Sets](sets-and-deltas.md#deployment-sets).
+
+- **Improved:** Performance of the `All deployment versions` functionality in the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview) for applications with a large number of past deployments.
+ 
+- **Fixed:** Image version selector no longer requests all versions even if `show archived` option wasn't checked. It caused performance issues for images with a large number of archived versions.
+
+- **Fixed:** Long [Deploy comments](https://docs.humanitec.com/getting-started/create-sample-app#deploy-the-application) no longer affect accessibility and alignment of action buttons in active deployment cards on the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview).
+
+- **Fixed:** An issue related to failing diff of a past deployment after adding a shared resource.
```

Comment:
We could link to the new shared resources documentation here: https://docs.humanitec.com/using-humanitec/work-with-apps/shared-resources as well as diff deployment: https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments

Reply:
I've also capitalized Shared Resource and Deployment Diff, let me know if that's fine or not 😊

Reply:
True yeah, we should adapt the Nomenclature of the docs team. Makes sense :-)